pkg/operator/ceohelpers/common.go

```diff
@@ -0,0 +1,49 @@
+package ceohelpers
+
+import (
+	"fmt"
+
+	"github.com/ghodss/yaml"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	corev1listers "k8s.io/client-go/listers/core/v1"
+)
+
+// ReadDesiredControlPlaneReplicasCount simply reds the desired Control Plane replica count from the cluster-config-v1 configmap in the kube-system namespace
+func ReadDesiredControlPlaneReplicasCount(configMapListerForKubeSystemNamespace corev1listers.ConfigMapNamespaceLister) (int, error) {
```

Comment:
@hasbro17  you have already reviewed this commit in https://github.com/openshift/cluster-etcd-operator/pull/767
I'll rebase once it merges